data/calgoncarbon

```diff
@@ -0,0 +1,6 @@
+calgon.cn
+calgon.com
```

Comment:
There are multiple countries domains listed in [calgon.com](https://www.calgon.com/). Please take some time to add them.

Reply:
> There are multiple countries domains listed in [calgon.com](https://www.calgon.com/). Please take some time to add them.

Done. Thanks to remind me.